Doppler.Jobs.Server/Startup.cs

```diff
@@ -74,6 +75,11 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddTransient<DopplerSapService>();
 
+            var connectionString = Configuration.GetConnectionString("DopplerDatabase");
+            services.AddSingleton(connectionString);
```

Comment:
What is the type of `connectionString`?

Reply:
Could be better use the already declared `DopplerRepositorySettings` class that hold the `ConnectionString` property. And use `IOption<T>` pattern could be improve this code.

Reply:
> What is the type of `connectionString`?

I had an error between different approach, I fixed the problem with DopplerRepositorySettings class thanks @elsupergomez